tests/functional/tests/bundles/ceph.yaml

```diff
@@ -17,6 +17,8 @@ machines:
     constraints: cores=1 mem=2G root-disk=20G
   '5':
     constraints: cores=1 mem=2G root-disk=20G
+  '6':
+    series: focal
```

Comment:
Can we define minimal `constraints` for this machine?

Reply:
Is it needed for running 3 `ceph-mon` units? I did not experience any issues during testing

Reply:
I think it's a good practice to avoid using too big machine (draw resources) or to small machine (tests will take too long).